test/Producers/MixpanelEventsProducerTest.php

```diff
@@ -101,4 +101,40 @@ public function testCreateAliasRespectsConsumerSetting() {
 
         unlink($tmp_file);
     }
+
+    public function testIdentifyInvalidAnonId() {
+        $user_id = 1;
+        $anon_id = 111;
+        try {
+            $test = $this->_instance->identify($user_id, $anon_id);
+        } catch (Exception $e) {
+            $this->assertEquals($e->getMessage(), 'Tried to identify but anon_id is not in UUID v4 format');
```

Comment:
won't this test still pass if no exception is thrown?

Reply:
you're right, i'm going to modify this test using @expectedException annotation